src/storage/backup/src/lib.rs

```diff
@@ -48,9 +48,9 @@ pub type MetaBackupJobId = u64;
 pub struct MetaSnapshotMetadata {
     pub id: MetaSnapshotId,
     pub hummock_version_id: HummockVersionId,
-    /// It actually stores object id.
-    /// Bad naming, which won't be changed due to compatibility issue.
-    pub ssts: HashSet<HummockSstableObjectId>,
+    // rename to `ssts` for backward compatibility
+    #[serde(rename = "ssts")]
+    pub objects: HashSet<HummockRawObjectId>,
```

Comment:
Will HummockRawObjectId be serialized here to integer like before? For forward-compatibility concerns, i.e. older RW version need be able to read meta snapshot created by newer RW version's.

Reply:
Yes. The `TypedPrimitive` impl serde as the same as the inner primitive type
```
impl<const C: usize, P: Serialize> Serialize for TypedPrimitive<C, P> {
    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
    where
        S: Serializer,
    {
        self.0.serialize(serializer)
    }
}

impl<'de, const C: usize, P: Deserialize<'de>> Deserialize<'de> for TypedPrimitive<C, P> {
    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
    where
        D: Deserializer<'de>,
    {
        Ok(Self(<P as Deserialize>::deserialize(deserializer)?))
    }
}
```